dev/devicelab/bin/tasks/flutter_gallery_ios__start_up_xcode_debug.dart

```diff
@@ -0,0 +1,16 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_devicelab/framework/devices.dart';
+import 'package:flutter_devicelab/framework/framework.dart';
+import 'package:flutter_devicelab/tasks/perf_tests.dart';
+
+Future<void> main() async {
+  deviceOperatingSystem = DeviceOperatingSystem.ios;
+  await task(createFlutterGalleryStartupTest(
+    runEnvironment: <String, String>{
+      'FORCE_XCODE_DEBUG': 'true',
```

Comment:
So which of the new workflows will this test?

Reply:
This will test `--use-application-binary` with Xcode being the one to install and launch the app

Reply:
Ahh right, so without this env var, it will use `ios-deploy`, right?

So this means we are not yet testing the debugging workflow, and doing so in the devicelab is blocked by having physical machines, is that right?

Reply:
Yes, until we put an iOS 17 physical device with Xcode 15 in the devicelab, it won't test the new workflows. 

So I added this environment variable to temporarily force it to test the new Xcode debugging workflow - where the project is opened and automated to run through Xcode interface. Note: This won't work with the `devicectl` workflows because that's only included in Xcode 15.

Reply:
acknowledged :+1: